app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".FurnitureActivity"></activity>
+        <activity android:name=".FurnitureActivity" />
```

Comment:
Fix the XML tag has empty body error as explained here:
https://stackoverflow.com/questions/58778349/xml-tag-has-empty-body-no-default-activity